types/service-worker-mock/service-worker-mock-tests.ts

```diff
@@ -0,0 +1,18 @@
+import makeServiceWorkerEnv from 'service-worker-mock';
```

Comment:
nit: For correctness, I prefer to use `import makeServiceWorkerEnv = require('service-worker-mock');` instead and remove the flag `esModuleInterop`. And to clarify, both of the methods are correct. I just prefer this method as I prefer not to enable flags.